src/ui/server/controllers/insurance/eligibility/companies-house-number/index.ts

```diff
@@ -18,9 +18,10 @@ export const FIELD_ID = FIELD_IDS.INSURANCE.ELIGIBILITY.HAS_COMPANIES_HOUSE_NUMB
 export const PAGE_VARIABLES = {
   FIELD_ID,
   PAGE_CONTENT_STRINGS: PAGES.INSURANCE.ELIGIBILITY.HAS_COMPANIES_HOUSE_NUMBER,
+  HAS_SAVE_AND_BACK: false,
```

Comment:
Is this needed? If there is save and back functionality, `SAVE_AND_BACK_URL` would be defined?